packages/provider/src/api/authMiddleware.ts

```diff
@@ -46,12 +46,14 @@ const extractHeaders = (req: Request) => {
 
     if (!signature || !blocknumber) {
         throw new ProsopoApiError('CONTRACT.INVALID_DATA_FORMAT', {
-            context: { error: 'Missing signature or block number' },
+            context: { error: 'Missing signature or block number', code: 400 },
```

Comment:
you've used status on other lines, is it status or code?

Reply:
Should be code everywhere. Status incorrectly used in Flux error handler. Good catch